src/customConfigTests/src/test/kotlin/org/vaccineimpact/orderlyweb/customConfigTests/SeleniumDebugHelper.kt

```diff
@@ -1,31 +1,35 @@
 package org.vaccineimpact.orderlyweb.customConfigTests
 
-import org.junit.rules.TestWatcher
-import org.junit.runner.Description
+import org.junit.jupiter.api.extension.ExtensionContext
+import org.junit.jupiter.api.extension.TestWatcher
 import org.openqa.selenium.By
 import org.openqa.selenium.WebDriver
 import org.openqa.selenium.logging.LogType
 
-class SeleniumDebugHelper: TestWatcher()
+class SeleniumDebugHelper: TestWatcher
```

Comment:
Can we lose the class member `driver` if all usages of driver are now coming from the contexts?

Reply:
Oh yeah, good spot, thanks